test/spec/trackingModules/SelectionTracking.spec.js

```diff
@@ -0,0 +1,53 @@
+import {
+  bootstrapModeler,
+  getBpmnJS,
+  inject
+} from 'bpmn-js/test/helper';
+
+import { injectStyles } from 'test/TestHelper';
+
+import BpmnJSTracking from 'src/BpmnJSTracking';
+
+
+describe('SelectionTracking', function() {
+
+  const diagram = require('test/spec/simple.bpmn').default;
+  let bpmnJSTracking;
+
+  injectStyles();
```

Comment:
Typically I'd expect styles to be injected once per the whole test suite. Note that the styles will be overlapping if we do it in each spec.js file.